pom.xml

```diff
@@ -23,4 +23,16 @@
     <module>java</module>
   </modules>
 
+  <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>animal-sniffer-maven-plugin</artifactId>
+          <version>1.17</version>
```

Comment:
Move this to pluginManagement section to [java/pom.xml](https://github.com/googlei18n/libphonenumber/blob/master/java/pom.xml) as it is the right place,  because:
- Artifiact "libphonenumber-parent" is been used as parent by all child components (tightly coupled and cannot exist without parent) and it contains the plugin management for them. 
- Artifact "libphonenumber-build-parent" is only used for building purpose in GitHub but it is not a individual binary as per our [maven builds](http://repo1.maven.org/maven2/com/googlecode/libphonenumber/).